molecule/default/tests/test_default.py

```diff
@@ -29,3 +37,30 @@ def test_files(host, f):
 def test_files2(host, f):
     """Test that the expected files and directories were deleted."""
     assert not host.file(f).exists
+
+
+def test_version_and_license(host):
+    """Verify that Cobalt Strike is licensed and is an expected version."""
+    cmd = host.run("cd /opt/cobaltstrike && ./teamserver")
+    regex = r"^\[\*\] Team Server Version: (?P<version>\d+\.\d+\.\d+) \((?P<date>\d{8})\) (?P<licensed>.*)$"
+    # Note that re.MULTILINE is critical here, since the output of the
+    # command spans several lines
+    print(repr(strip_ansi(cmd.stdout)))
+    match = re.search(regex, strip_ansi(cmd.stdout), re.MULTILINE)
+    assert match is not None, "Regex does not match Cobalt Strike output."
+
+    version = match.group("version")
+    assert (
+        semver.VersionInfo.parse(version) >= min_expected_version
+    ), f"Cobalt Strike version is expected to be greater than or equal to {min_expected_version}."
+
+    date = match.group("date")
+    year = int(date[0:4])
+    month = int(date[4:6])
+    day = int(date[6:8])
```

Comment:
So magical!  🪄 
How about nested named capture groups within the date?

Reply:
Please see commit 361e1df.

Reply:
<h1>🌶️ Your RegEx is now En Fuego!  🌶️ </h1>

![wrestling](https://user-images.githubusercontent.com/3229435/197799000-a2499a94-a845-4f32-a471-a79166fbb715.gif)
<h1>¡Sensacional!</h1>